webpack.common.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    entry: './src/index.js',
+    plugins: [
+        new CleanWebpackPlugin(),
+        new HtmlWebpackPlugin({
+            filename: 'index.html',
+            inject: true,
+            template: path.resolve(__dirname, 'public', 'index.html'),
+        }),
+    ],
+    module: {
+        rules: [
```

Comment:
As for now it's not necessary, but it's pretty basic to have simple css support. You could use 'style-loader' (for dev), MiniCssExtractPlugin.loader (for prod), 'css-loader'